dashboard/app/views/levels/editors/_netsim.html.haml

```diff
@@ -10,11 +10,15 @@
     %tr
       %td= boolean_check_box f, :show_routers_in_lobby
       %td= f.label :show_routers_in_lobby, 'Show Routers/Rooms in Lobby'
-      %td Whether router nodes (or broadcast rooms) should appear in the lobby list at all.
+      %td
+        Whether router nodes (or broadcast rooms) should appear in the
+        lobby list at all.
```

Comment:
Lots of change in this file because I learned the trick to wrapping long lines.  I'll comment pertinent functional changes.